docs/new/src/pages/ecosystem/index.js

```diff
@@ -148,9 +178,8 @@ const EcosystemIndex = (props) => {
             gap: 20,
           }}
         >
-          {sortedPages.map((id) => {
+          {filteredPages.map((id) => {
```

Comment:
Having played with this, an empty state would be nice. Something like _"No integrations found for 'enyov'"_ so you're given a hint that your input has a typo...?

I have also briefly been wondering about debouncing the user input, but from playing with it, I don't think there's a need for it.

Reply:
ahh yeah, I can add that.

Reply:
Have done so in df76546187a17832af119e750ed30dbfcd5c967f